include/Result.hpp

```diff
@@ -0,0 +1,27 @@
+//
+// Created by Tianyi Wang on 5/14/23.
+//
+
+#ifndef DD_EVAL_RESULT_HPP
+#define DD_EVAL_RESULT_HPP
+
+#include "ResultBase.hpp"
+
+#include <nlohmann/json.hpp>
+#include <string>
+
+using json = nlohmann::json;
+
+class Result : public ResultBase {
+public:
+  Result(const std::string& name, double value);
+
+  json        toJson() const override;
+  std::string toString() const override;
+
+private:
+  std::string name_;
+  double      value_;
```

Comment:
As pointed out by the clang-tidy check, these do not follow the naming conventions. (do not use trailing underscores for member variables)